### PR TITLE
[#10041] Request token price after adding

### LINKED ISSUE #10041: Fiat value is '0' in added custom tokens
# Bug Report

## Problem

When adding a custom token with total amount in fiat > 0, we don't calculate fiat value (only token amount is displayed)

#### Expected behavior

Fiat value calculated in added token with valid > 0 amount (until app restart)

#### Actual behavior

Fiat value is '0' in added token with amount until app restart
Screenshot 2020-02-14 at 11 37 39

### Reproduction
- Open Status and recover an account with custom tokens available (or new account - and you may add watch-only address then)
- In `Wallet` -> `...` -> `Manage assets` -> `Add custom token` paste address of custom token (e.g. 0x016ee7373248a80BDe1fD6bAA001311d233b3CFa  which is `3X Short Bitcoin`)
- Add watch-only address having that token (e.g. `0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be`) 

### Additional Information

- Status version: `1.0.1`
- Operating System: Android and iOS

### PR DESCRIPTION
**Pull request by rasom**
fix #10041 

status: ready

### CODE REVIEW

File: src/status_im/utils/prices.cljs [hunk 1]
```diff
@@ -40,6 +41,11 @@
                                      :last-day (:OPEN24HOUR entry)}}))}))))
 
 (defn get-prices [from to mainnet? on-success on-error chaos-mode?]
+  (log/debug "[prices] get-prices"
+             "from" from
+             "to" to
+             "mainnet?" mainnet?
+             "chaos-mode?" chaos-mode?)
```
Comment (yenda) [inline src/status_im/utils/prices.cljs:48]: didn't we remove chaos-mode?

**Review by yenda — APPROVED** [review]

File: src/status_im/utils/prices.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/utils/prices.cljs:48]: I have no idea, it is at least in this function

Comment (rasom) [reply src/status_im/utils/prices.cljs:48]: @yenda as far as I see it is still in the project

**Comment by flexsurfer** [comment]
@rasom related bug, when we remove token we don't recalculate total value in fiat, probably could be done in this PR ?

File: src/status_im/utils/prices.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/utils/prices.cljs:48]: i just disabled it, not removed, but dunno if we want to remove all related code completely

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**